packages/frontend/src/main/components/common/SpeckleViewer.vue

```diff
@@ -89,7 +89,7 @@ export default {
 <style>
 #rendererparent {
   position: relative;
-  display: inline-block;
+  display: block;
```

Comment:
Fix for vertical scrollbar in viewer https://discord.com/channels/726756379083145269/850367221482258463/984774757004230696